### ISSUE TITLE
minor typo in itty.py

### ISSUE THREAD

**Issue by marly** [toastdriven/itty#1]
Noticed this while browsing the source. Line 444 should be:

```
raise RuntimeError("Server '%s' is not a valid server. Please choose a different server." % server)
```

**Comment by toastdriven** [comment]
Sorry for the horrifically slow response. I never got a notification of this issue. I'll try to address it within the next week or so.

**Comment by toastdriven** [comment]
Fixed in SHA: e5b0cc396e820e0fb3754ea62f0f79e862234b05
